### PR TITLE
Update the documentation

### PR DESCRIPTION
**Pull request by nicomiguelino**
#### Overview

- Fixes Screenly/Anthias#1807 &mdash; The docs need to be updated to remain relevant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +15,16 @@ Want to help Anthias thrive? Support us using [GitHub Sponsor](https://github.co
...
 
-## Disk images
+## Balena disk images
 
 The quickest way to get started is to use [Raspberry Pi Imager](https://www.screenly.io/blog/2022/12/13/anthias-and-screenly-now-in-rpi-imager/), where you can find Anthias under `Other specific-purpose OS`. Alternatively, you can find our pre-built disk images (powered by [Balena Hub](https://hub.balena.io/)) [here](https://github.com/Screenly/Anthias/releases/latest/).
 
-Do however note that that we are still in the process of knocking out some bugs. You can track the known issues [here](https://github.com/Screenly/Anthias/projects/8).
+Do however note that that we are still in the process of knocking out some bugs. You can track the known issues [here](https://github.com/Screenly/Anthias/issues). You can also check the discussions in the [Anthias forums][1].
+
+If you'd like more control over your digital signage isntance, try installing it on
```
Comment (vpetersson) [inline README.md:24]: Typo.

**Comment by vpetersson** [comment]
Nice update! Let's remove the old docs too but looks good to me.

File: docs/developer-documentation.md [hunk 2]
```diff
@@ -1,135 +1,179 @@
...
+* **Redis** (`redis`) is used as a database, cache and message broker.
+* The **database** component uses **SQLite** for storing the assets information.
+
+There are currently three major branches in the repository:
+
+| Version       | Branch      | Comment    |
+| :------------ | :---------- | :--------- |
+|  Developer    | [`master`](https://github.com/Screenly/Anthias) | This is where we test things and apply latest fixes. |
+|  Production   | [`production`](https://github.com/Screenly/Anthias/tree/production) | This is the branch disk images are built from and should be properly tested |
+|  Experimental | [`experimental`](https://github.com/Screenly/Anthias/tree/experimental) | This is the branch for experimenting, such as using a new web browser |
```
Comment (vpetersson) [inline docs/developer-documentation.md:25]: This needs to be overhauled. The `experimental` branch has been abandoned, and we might consider deprecating `production` too in order to move to rolling releases instead.

**Comment by nicomiguelino** [comment]
Hello, @vpetersson. Requesting another review... Thanks!

File: docs/README.md [hunk 3]
```diff
@@ -1,43 +1,92 @@
...
+Run the following command in your console:
+
+```bash
+$ bash <(curl -sL https://install-anthias.srly.io)
+```
+
+Alternatively, you can also run the following command:
+
+```bash
+$ /home/${USER}/screenly/bin/run_upgrade.sh
```
Comment (vpetersson) [inline docs/README.md:73]: Can be simplified to '$HOME/screenly/bin/run_upgrade.sh'

File: docs/README.md [hunk 4]
```diff
@@ -1,43 +1,92 @@
...
 
 The SQLite Database can be found here: `~/.screenly/screenly.db`
 
-Its a SQLite Database and can be modified with the sqlite3 CLI. The schema is relatively straight forward if you are a developer; the columns of most interest to you will be `name` and `is_enabled`. In addition `start_date` is useful if you want to use this in a disconnected manner.
+Its a SQLite Database and can be modified with the `sqlite3` CLI. The schema is relatively straightforward if you're already familiar. The columns of most interest to you will be `name` and `is_enabled`. In addition, `start_date` is useful if you want to use this in a disconnected manner.
 
-# Wi-Fi Setup
+## Wi-Fi Setup
 
 - Read the [Wi-Fi Setup](wifi-setup.md) page for more details on how to set up Wi-Fi on the Raspberry Pi.
```
Comment (vpetersson) [inline docs/README.md:92]: This Wi-Fi page probably also needs an overhaul.

**Review by vpetersson — APPROVED** [review]
Minor nitpick. Good work.

File: docs/README.md (hunk 4 above)
Comment (nicomiguelino) [reply docs/README.md:92]: Done. Thanks!
(Changes can be seen [here](https://github.com/Screenly/Anthias/pull/1816/files#diff-4ef6a9c8e5b4796726e15b6a192e7c2fde11bac37841e371c37337d94c4fec12).)

**Pull request merged**